_deferred.yml

```diff
@@ -17,4 +17,7 @@ apps:
       NOTIFICATIONS_QUEUE: XXX
       PUB_SUB_QUEUE: XXX
       LOGS_DIR: XXX
-      LOGS_LEVEL: XXX
\ No newline at end of file
+      LOGS_LEVEL: XXX
```

Comment:
Hi @ricardoul, what's the purpose of the "XXX"? Are these not supposed to be default values?